### PR TITLE
SDI-15 Create visit nomis endpoint - sort event_id

### PR DESCRIPTION
**Pull request by steverendell**

### CODE REVIEW

File: src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/service/VisitService.kt [hunk 1]
```diff
@@ -160,16 +162,27 @@ class VisitService(
...
+        eventId = getNextEvent()
+      )
     )
 
     visitDto.visitorPersonIds.forEach {
       val person = personRepository.findById(it).orElseThrow(DataNotFoundException("Person with id=$it does not exist"))
       visit.visitors.add(
-        VisitVisitor(visit = visit, person = person, eventStatus = scheduledEventStatus)
+        VisitVisitor(visit = visit, person = person, eventStatus = scheduledEventStatus, eventId = getNextEvent())
       )
```
Comment (andymarke) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/service/VisitService.kt:177]: do you think you need it on each visitor as well as the dummy row?

**Review by andymarke — APPROVED** [review]
LGTM

File: src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/service/VisitService.kt [hunk 2]
```diff
@@ -160,16 +162,27 @@ class VisitService(
...
     }
   }
+
+  private fun getNextEvent(): Long {
+    // TODO Reviewing with Paul M as to whether this is a wise course to take!
+    // The stored proc uses event_id.nextval
+    return visitVisitorRepository.getEventId()
```
Comment (steverendell) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/service/VisitService.kt:184]: Actually my current thinking is to just go with this so Ive not asked Paul yet

File: src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/service/VisitService.kt (hunk 1 above)
Comment (steverendell) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/service/VisitService.kt:177]: Ah good point - no, I think actually only needed on dummy row , will refine that next time

**Pull request merged**